_posts/2023-10-10-endless-os.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: distro
+author: "Sam Hewitt"
+updater: "cassidyjames"
+title: "Endless OS"
+date: "2013-09-14"
+updated: "2023-10-10"
+logo: "endlessos"
+image: "endlessos-5.png"
+caption: "Endless OS 5 with its custom GNOME desktop"
+tags: [desktop]
+category: [beginner]
+notes:
+  homepage: https://endlessos.org/os
+  version: "5.0.6"
+  based-on: ["Debian"]
+  desktops: [GNOME]
+  install: graphical
+  developers:
+    - name: Endless OS Foundation
+      url: https://endlessos.org/
+---
+
+Endless OS is a beginner-friendly distribution with an interface designed to "feel natural and intuitive", taking cues from mobile operating systems and presenting apps as a grid of icons right on main screen. It's designed so that no Internet is required, but also works well on Internet-connected computers.
```

Comment:
While we're at it :)

```suggestion
Endless OS is a beginner-friendly distribution with an interface designed to "feel natural and intuitive", taking cues from mobile operating systems and presenting apps as a grid of icons right on the main screen. It's designed so that no Internet is required, but also works well on Internet-connected computers.
```